test/org/opensolaris/opengrok/web/DirectoryListingTest.java

```diff
@@ -84,11 +85,14 @@ private void create() throws Exception {
             assertTrue("Failed to set modification time",
                        file.setLastModified(val));
 
-            if (size > 0) {
-                FileOutputStream out = new FileOutputStream(file);
-                byte[] buffer = new byte[size];
-                out.write(buffer);
-                out.close();
+            if (!"-".equals(size)) {
```

Comment:
Also, while you're at it, change create() to use mkdir() for the directories.and fix setUp()